cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -266,6 +278,15 @@ public void testNewToProposalReadyRemoveBroker(VertxTestContext context) throws
         this.krNewToProposalReady(context, CruiseControlEndpoints.REMOVE_BROKER, kr);
     }
 
+    /**
+     * See the {@link KafkaRebalanceAssemblyOperatorTest#testNewToProposalReadyRebalance} for description
```

Comment:
wrong javadoc? rebalance is not related to it.
the same applies to all the other following tests.

Reply:
@ppatierno ^^

Reply:
Yeah. Let's stick with that then. We are doing the same for add/remove brokers tests as well.